### DISCUSSION TITLE
Teleport Connect client and Application Access

### DISCUSSION THREAD

**Discussion by Aloysius-AS** [gravitational/teleport#39482] [Q&A] [answered]
Hi!

I am trying to evaluate Teleport Connect client and how user-friendly is it.

When a Teleport Application (web application) is launched via Connect, I would be prompted to login to Teleport in a web browser. Is there a way to directly access the Teleport Application, similarly to the process flow when logging into Teleport through web browser?

Accessing Teleport Application via Connect would require me login twice:
- when accessing Connect
- when accessing Teleport Application for the first time in the launched browser

It would be nice if there is a way to reduce the number of login to just on Teleport Connect.

I will be very grateful for detailed answer.

**Comment by zmb3** [comment, marked as answer]
At the moment, you have to be logged in to the web UI. If you are already logged in and have a valid session, you won't be prompted to log in again.

There is not currently any way to propagate your existing Teleport Connect session to the web UI, as these sessions are managed differently (Teleport Connect sessions are backed by client-certificates, whereas web UI sessions are backed by secure cookies).
